tools/benchmark.py

```diff
@@ -89,6 +89,9 @@ def strace_parse(summary_text):
     summary = {}
     # clear empty lines
     lines = list(filter(lambda x: x and x != "\n", summary_text.split("\n")))
+    # filter out non-relevant lines. See the error log at
+    # https://github.com/denoland/deno/pull/3715/checks?check_run_id=397365887
+    lines = [x for x in lines if x.find("detached") == -1]
```

Comment:
Ah! Thank you!

Reply:
Landed separately here https://github.com/denoland/deno/commit/e83658138bff3605bd37c2b4ae4703081d884729